CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.19.0
+
+This release adds a new SDK, [@sentry/opentelemetry-node](./packages/opentelemetry-node/),
+which is available as an alpha release to integrate OpenTelemetry performance tracing with Sentry.
+Give it a try and let us know if you have any feedback or problems with using it. (#6000)
+
+ ### Deprecation for `tracingOrigins` added
+
+ The `tracingOrigins` option is deprecated in favor of using `shouldCreateSpanForRequest` and `tracePropagationTargets`.
```

Comment:
l: I think we can shorten this a little but no strong feelings if we leave it as is
```suggestion
This release also deprecates the `tracingOrigins` option in favor of using `shouldCreateSpanForRequest` and `tracePropagationTargets`.
```